tests/testFunction.py

```diff
@@ -628,42 +628,30 @@ def testDFuncDParameters(self):
 
             self.assertAlmostEqual(f(x, y), sum([params[i]*dFdC[i] for i in range(len(params))]))
 
-    @unittest.skip("temporary skip while wrapping")
-    def testCast(self):
+    def testAutoCast(self):
```

Comment:
In the past we have simply removed the test and assume that this is covered by pybind11's own unit tests.